holoviews/plotting/bokeh/renderer.py

```diff
@@ -31,6 +30,28 @@
 </script>
 """
 
+bokeh_msg_handler = """
+var plot = Bokeh.index["{plot_id}"];
```

Comment:
This is the block of JS that becomes the body of the message handler callback? I might add a comment if so...